pkg/api/v20191231preview/openshiftcluster_validatestatic_test.go

```diff
@@ -51,7 +50,7 @@ func validOpenShiftCluster() *OpenShiftCluster {
 			ClusterProfile: ClusterProfile{
 				PullSecret:      `{"auths":{"registry.connect.redhat.com":{"auth":""},"registry.redhat.io":{"auth":""}}}`,
 				Domain:          "cluster.location.aroapp.io",
-				Version:         version.DefaultInstallStream.Version.String(),
+				Version:         "4.10.0",
```

Comment:
Hey Matt, why are we looking at 4.10 specifically here?

Reply:
No real reason, I just needed _some_ version to plug into `ClusterProfile` structs.